tests/get_total_coverage.sh

```diff
@@ -20,13 +20,13 @@ tox -e py26,py27,py36,py37 -- --cov-append
 
```

Comment:
Couldn't find this file referenced anywhere. Do you know where it gets called from?

Reply:
This is only called manually. It needs to be integrated into a CI or test-harness needs to be changed to call `get_coverage.sh` or something else to create coverage data. But nothing happened in this direction, yet.

Reply:
Ah right, should be ok to change the name then? I have changed all the references to the other coverage scripts it references (e.g. `get_coverage.sh`)

Reply:
Probably not needed but it does not hurt, too.